### PR TITLE
Add UI test for editing time entry description

### LINKED ISSUE #4443: EditTimeEntry: Edit time entry description
Sign in
Start time entry
Enter description
Save Entry
Stop Entry
Tap on entry
Edit entry description
Confirm/Save

### PR DESCRIPTION
**Pull request by arturs105**
### Relationships

Closes #4443

## Review hints
Running the test is recommended.

## :squid: Permissions
🦑

### CODE REVIEW

File: Toggl.Giskard.Tests.UI/Configuration.cs [hunk 1]
```diff
@@ -8,7 +8,6 @@ public static class Configuration
         public static AndroidApp GetApp()
             => ConfigureApp
                 .Android
-                .ApkFile("../../bin/Release/com.toggl.giskard.apk")
```
Comment (khoi) [inline Toggl.Giskard.Tests.UI/Configuration.cs]: 👀

File: Toggl.Tests.UI/EditTimeEntryTests.cs [hunk 2]
```diff
@@ -1,14 +1,48 @@
...
+
+            app.WaitForElement(EditTimeEntry.EditDescription);
+            app.Tap(EditTimeEntry.EditDescription);
+            app.EnterText(appendedText);
+            app.DismissKeyboard();
+
+            app.Tap(EditTimeEntry.Confirm);
+            //On Android the confirm button needs to be tapped twice.
+            //Once to finish editing the description and once more to close the edit view
+            if (app.Query(EditTimeEntry.Confirm).Length > 0)
```
Comment (khoi) [inline Toggl.Tests.UI/EditTimeEntryTests.cs]: I'd put this into  `EditTimeEntryExtensions`'s `ConfirmEditTimeEntry` method, and then Giskard implementation can tap the button twice there. Similar the current `StartTimeEntryExtensions`

File: Toggl.Giskard.Tests.UI/Configuration.cs (hunk 1 above)
Comment (arturs105) [reply Toggl.Giskard.Tests.UI/Configuration.cs]: We shouldn't run tests on Release config.

Comment (amulware) [reply Toggl.Giskard.Tests.UI/Configuration.cs]: Please be aware of Release vs Release.PlayStore

Comment (khoi) [reply Toggl.Giskard.Tests.UI/Configuration.cs]: I don't think this belongs to the PR though

**Pull request merged**